content/topics/kotlin/properties-and-fields/_index.md

```diff
@@ -141,3 +141,42 @@ Accessing a lateinit property before it has been initialized throws a special ex
 The most common kind of properties simply reads from (and maybe writes to) a backing field. On the other hand, with custom getters and setters one can implement any behaviour of a property. Somewhere in between, there are certain common patterns of how a property may work. A few examples: lazy values, reading from a map by a given key, accessing a database, notifying listener on access, etc.
 
 Such common behaviours can be implemented as libraries using delegated properties.
+
+### Classes and Interfaces
```

Comment:
Heading on line 29 is broken and functions should be written as inline code

Reply:
Fixed